dev/service-registry/client-core/src/main/scala/com/lightbend/lagom/internal/registry/ServiceRegistryClient.scala

```diff
@@ -10,7 +10,8 @@ import scala.collection.immutable
 import scala.concurrent.Future
 
 private[lagom] trait ServiceRegistryClient {
-  def locateAll(serviceName: String): Future[immutable.Seq[URI]]
+  // TODO: add support for `protocol:Option[String]` on lookup
+  def locateAll(serviceName: String, portName: Option[String]): Future[immutable.Seq[URI]]
```

Comment:
Eventually we'll need to also add the `protocol` field but since we're likely to stay on `tcp` only I waited.

Reply:
Maybe it could reuse the [`Lookup`](https://github.com/akka/akka-management/blob/f919c114f0d2845074a409056f8304a508323aa0/discovery/src/main/scala/akka/discovery/SimpleServiceDiscovery.scala#L79-L100) class from Akka Discovery.

Reply:
Sure! didn't want the `Lookup` class to take over so early but we're definitely going that direction.

Reply:
Something for later, anyway.